code-reviews/faq/activation-codes.md

```diff
@@ -0,0 +1,55 @@
+---
+title: How to use activation codes
+description: Learn how to redeem activation codes and add credits to your kluster.ai account
+---
+
+# How to use activation codes
+
+Activation codes provide promotional credits for [kluster.ai](https://www.kluster.ai/){target=_blank} plans. This guide shows you how to redeem your activation code and apply it to your subscription.
+
+## Prerequisites
+
+Before redeeming an activation code, ensure you have:
+
+- A kluster.ai account. If you don't have one, [create an account](https://platform.kluster.ai/signup){target=_blank}.
+- An activation code that matches the plan you want to subscribe to.
+
+## Redeeming your activation code
+
+Follow these steps to redeem your activation code:
+
+1. Log in to your [kluster.ai account](https://platform.kluster.ai){target=_blank}. First-time users receive 50 free code reviews under the trial plan.
+
+2. Click the **upgrade** button in the top right corner of the platform.
+
+    ![Upgrade button on kluster.ai platform](/images/code-reviews/code/faq/activation-codes/activation-code-1.webp)
+
+3. On the plans page, choose the subscription plan you want. Make sure your activation code matches the plan you select, then click **Subscribe**.
+
+    ![kluster.ai subscription plans](/images/code-reviews/code/faq/activation-codes/activation-code-2.webp)
+
+    You'll be redirected to the Stripe checkout page.
+
+4. On the Stripe checkout page, click the **Add promotion code** section and paste your activation code.
+
+    ![Stripe checkout with promotion code field](/images/code-reviews/code/faq/activation-codes/activation-code-3.webp)
+
+5. Once you apply the code, Stripe will validate it and show the discount applied. The total due today will reflect the promotional credit.
+
+    ![Activation code applied successfully](/images/code-reviews/code/faq/activation-codes/activation-code-4.webp)
+
+    Complete the checkout process by clicking **Subscribe**. Your promotional credits will be immediately applied to your account.
+
+## Troubleshooting
+
+### Code not working
+
+If your activation code doesn't work:
+
+- Verify the code matches your selected plan. Activation codes are plan-specific (Starter, Team, or Enterprise).
+- Check that the code hasn't expired.
+- Ensure you haven't already used this code on your account.
+
+## Need help?
+
+If you encounter any issues with activation codes, reach out to our support team or join our [Discord community](https://discord.com/invite/klusterai){target=_blank}.
```

Comment:
Do you want to say reach out to our support team on Discord? I think they are same thing, unless there is a support email or contact, in which case I would put that as a link under support team.